packages/sources/coinpaprika/src/config.ts

```diff
@@ -8,9 +8,16 @@ export const DEFAULT_API_ENDPOINT = 'https://api.coinpaprika.com'
 
 export const makeConfig = (prefix?: string): Config => {
   const config = Requester.getDefaultConfig(prefix)
+  const headers: { [T: string]: string | boolean } = {}
+  if (config.apiKey) headers['Authorization'] = config.apiKey
+
+  const isInTestMode = util.getEnv('IS_TEST_MODE', prefix)
```

Comment:
We should use parseBool here, otherwise a false-y value could be treated as true here